python/pyspark/mllib/clustering.py

```diff
@@ -103,10 +117,11 @@ def load(cls, sc, path):
 class KMeans(object):
 
     @classmethod
-    def train(cls, rdd, k, maxIterations=100, runs=1, initializationMode="k-means||", seed=None):
+    def train(cls, rdd, k, maxIterations=100, runs=1, initializationMode="k-means||",
+              initializationSteps=5, epsilon=1e-4, seed=None):
```

Comment:
This breaks compatibility. We didn't force keyword arguments and hence we have to keep the existing ordering of arguments.